hugegraph-pd/hg-pd-core/src/main/java/org/apache/hugegraph/pd/raft/RaftEngine.java

```diff
@@ -130,14 +136,35 @@ public boolean init(PDConfig.Raft config) {
     /**
      * Create a Raft RPC Server for communication between PDs
      */
-    private RpcServer createRaftRpcServer(String raftAddr) {
+    private RpcServer createRaftRpcServer(String raftAddr, List<PeerId> peers) {
         Endpoint endpoint = JRaftUtils.getEndPoint(raftAddr);
         RpcServer rpcServer = RaftRpcServerFactory.createRaftRpcServer(endpoint);
+        configureRaftServerIpWhitelist(peers, rpcServer);
```

Comment:
How does the raft group member change?

Reply:
1、The whitelist IPs are sourced from the peer-list configuration file. The current solution rejects all Raft-related requests from nodes not in the peer-list.
2、I understand that the members of the Raft group are a subset of the peer-list. Please correct me if there are any errors in this understanding.
thanks